src/ExternalEpic/canRender.ts

```diff
@@ -0,0 +1,26 @@
+import { BrazeMessageProps } from './index';
```

Comment:
I guess we'll be able to get rid of `ExternalEpic` once we've moved over to using the new component from this lib in DCR?

Reply:
Yes I think so. Although I was wondering if there is anyway to setup the Chromatic tests to compare the two components?

Reply:
Ooh that's an interesting thought. I'm not sure - Chromatic generally compares an accepted baseline version of a component with a later iteration of the same component. Would be cool to spend some minutes on figuring out what's possible. This could be a really cool way of avoiding drift between the two components.